BDArmory/UI/BDArmorySetup.cs

```diff
@@ -2953,7 +2953,7 @@ void WindowSettings(int windowID)
                         GUI.Label(SLeftSliderRect(++line, 1), $"{StringUtils.Localize("#LOC_BDArmory_Settings_TimeScale")}; ({BDArmorySettings.TIME_SCALE:G2}x)", leftLabel);
                         if (BDArmorySettings.TIME_SCALE != (BDArmorySettings.TIME_SCALE = BDAMath.RoundToUnit(GUI.HorizontalSlider(SRightSliderRect(line), BDArmorySettings.TIME_SCALE, 0f, BDArmorySettings.TIME_SCALE_MAX), BDArmorySettings.TIME_SCALE > 5f ? 1f : 0.1f)))
                         {
-                            Time.timeScale = BDArmorySettings.TIME_SCALE;
+                            if (!GameIsPaused) Time.timeScale = BDArmorySettings.TIME_SCALE;
```

Comment:
This will prevent the new time scale from being applied if the game is paused (either with the pause menu open or with the time scale being 0) until one toggles the Time Override off and on again. Is that intentional?

Reply:
No. That's vestigial, that missed getting removed from when I was experimenting around with trying to fix opening/closing the pause menu overriding the Time Override (Time.Timescale getting reset to 0/1, and needing the TO toggle toggled to reactivate the new scaling). Something for the ToDo list, but somewhat out-of-scope of the rest of the stuff in the PR.